src/v2/controllers/FormController.js

```diff
@@ -153,7 +153,9 @@ export default Controller.extend({
       // TODO: OKTA-243167 what's the approach to show spinner indicating API in flight?
       actionFn()
         .then((resp) => {
-          if (actionPath === 'cancel' && this.options.settings.get('useInteractionCodeFlow')) {
+          if (actionPath === 'cancel' && window.location.pathname === '/signin/forgot-password') {
```

Comment:
Would this be removed once backend change is merged as you indicated in Jira ? if yes then put a TODO here..